plasmapy/physics/dielectric.py

```diff
@@ -196,3 +199,40 @@ def cold_plasma_permittivity_LRP(B, species, n, omega):
         R += - omega_p ** 2 / (omega * (omega + omega_c))
         P += - omega_p ** 2 / omega ** 2
     return L, R, P
+
+# plasma permittivity in terms of Jackson function
+@u.quantity_input(omega=u.rad /u.s,
+                  kWave=u.rad /u.m,
+                  T=u.K,
+                  n=u.m**-3,
+                  z_mean=u.dimensionless_unscaled)
+def permittivity_1D_Maxwellian(omega,
+                               kWave,
+                               T,
+                               n,
+                               particle,
+                               z_mean=None):
+    """
+    omega [rad/s]
+    kWave [rad/m]
+    temperature [eV]
+    density [cm^-3]
+    """
```

Comment:
This is desperately in need of some better docstrings. I'd be happy to provide those if you'd like.

Reply:
Don't worry I'll clean it up. I've just been quite busy.